src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -777,11 +796,13 @@ export default class LazyMobXTable<T> extends React.Component<LazyMobXTableProps
 
     private getTable() {
         return (
-            <SimpleTable
-                headers={this.store.headers}
-                rows={this.store.rows}
-                className={this.props.className}
-            />
+            <div style={{overflowX: this.props.enableHorizontalScroll ? "auto" : "visible"}}>
```

Comment:
@alisman @adamabeshouse This is to conditionally enable horizontal scroll, not sure if this is the best way to do it though.

Reply:
seems reasonable to me.. not sure if its even worth it/better but could do this with conditional className instead of conditional style?